### PR TITLE
Fix issues in e2e tests

### PR DESCRIPTION
**Pull request by romasku**

### CODE REVIEW

File: neuro-cli/tests/e2e/test_e2e_share.py [hunk 1]
```diff
@@ -231,5 +231,5 @@ def test_add_grant_remove_role(request: Any, helper: Helper) -> None:
     captured = helper.run_cli(["acl", "list", "--full-uri", "-u", "public"])
     assert captured.err == ""
     result = [line.split() for line in captured.out.splitlines()]
-    assert [f"role://{role_name}", "read"] in result
+    assert [f"role://{role_name}", "read"] not in result
```
Comment (asvetlov) [inline neuro-cli/tests/e2e/test_e2e_share.py:234]: Did it ever work before?

File: neuro-cli/tests/e2e/test_e2e_storage.py [hunk 2]
```diff
@@ -116,7 +116,7 @@ def test_copy_local_file_to_platform_directory_explicit(
 
     helper.mkdir("folder", parents=True)
     # Upload local file to existing directory
-    helper.run_cli(["storage", "cp", "-t", helper.tmpstorage + "/folder", srcfile])
+    helper.run_cli(["storage", "cp", "-t", helper.tmpstorage + "folder", srcfile])
```
Comment (asvetlov) [inline neuro-cli/tests/e2e/test_e2e_storage.py:119]: Would you consider converting `helper.tmpstorage` from `str` to `URL` and applying `str(arg)` for all `run_cli()` arguments?
I believe, it makes tests more robust; `helper.tmpstorage / "folder"` is unambigous.

File: neuro-cli/tests/e2e/test_e2e_share.py (hunk 1 above)
Comment (romasku) [reply neuro-cli/tests/e2e/test_e2e_share.py:234]: Server behaviour changed recently. Now, if the role is removed, then we also remove permission to such a role.

File: neuro-cli/tests/e2e/test_e2e_storage.py (hunk 2 above)
Comment (romasku) [reply neuro-cli/tests/e2e/test_e2e_storage.py:119]: Good idea, thanks

**Comment by romasku** [comment]
Finally, it is green!!!
@asvetlov please re-review.

File: neuro-cli/tests/e2e/test_e2e_share.py (hunk 1 above)
Comment (asvetlov) [reply neuro-cli/tests/e2e/test_e2e_share.py:234]: Understood

**Review by asvetlov — APPROVED** [review]
LGTM!

**Pull request merged**